src/app/meetups/add-meetups/time-mask.directive.ts

```diff
@@ -0,0 +1,30 @@
+import { Directive, ElementRef, HostListener, Input } from '@angular/core';
+import { NgControl } from '@angular/forms';
+
+@Directive({
+  selector: 'input[planetTimeMask]'
+})
+export class TimeMaskDirective {
+
+  constructor(private _el: ElementRef) { }
+
+  @HostListener('input', [ '$event' ]) onInputChange(event) {
+    const inputValue = this._el.nativeElement.value;
+    console.log(inputValue);
```

Comment:
Please make sure to remove `console.log` before your final push.  If it's still a work in progress `console.log` is OK.